core/src/main/java/org/kohsuke/stapler/export/JSONDataWriter.java

```diff
@@ -74,6 +83,8 @@ protected void comma() throws IOException {
         needComma = true;
     }
 
+
+
```

Comment:
Inserting extra white space is inconsistent with the rest of the file.  Please remove it.
```suggestion
```